query/src/test/java/org/infinispan/query/backend/QueryInterceptorIndexingOperationsTest.java

```diff
@@ -151,7 +157,7 @@ public Entity1(String attribute) {
       }
    }
 
-   @Indexed(index = "theIndex")
+   @Indexed(index = "theIndex2")
```

Comment:
So it does not support multiple entities in the index anymore, or is this temporary?

Reply:
That's definitive.

Elasticsearch doesn't support multiple types in the same index either, for multiple reasons. Complexity is an obvious reason, but performance is another, as having multiple types in the same index increases the likeliness of sparse data, which apparently impairs the ability of Lucene to compress documents. See https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html#_why_are_mapping_types_being_removed

Reply:
Note you can still query multiple indexes in a single query, so feature-wise, nothing changes.